src/util/exception_utils.cpp

```diff
@@ -63,7 +63,6 @@ incorrect_goto_program_exceptiont::incorrect_goto_program_exceptiont(
 
 std::string incorrect_goto_program_exceptiont::what() const
 {
-  return message + " (at: " + source_location.as_string() + ")";
```

Comment:
Oh, my bad. Sorry about this one.

Reply:
It passed through code review without being spotted, so it hid well :-)